src/main/java/com/jalasoft/webservice/utils/FileManager.java

```diff
@@ -0,0 +1,44 @@
+package com.jalasoft.webservice.utils;
+
+/*
+ *
+ *  Copyright (c) 2019 Jalasoft.
```

Comment:
Please, could you add the following comments: class name, version information, date, and copyright notice (it is detailed at  https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141855.html -section 3. File Organization )

Reply:
Paolo give us this Copyright to configure it in our IDE. Also this page has a very old recommendation for Copyright.